gator/checks/check_CountCommandOutput.py

```diff
@@ -0,0 +1,77 @@
+"""Check that a command's output produces a specified amount."""
+
+import argparse
+
+from gator import checkers
+from gator import invoke
+
+
+def get_parser():
+    """Get a parser for the arguments provided on the command-line."""
+    # create the parser with the default help formatter
+    # use a new description since this is a stand-alone check
+    parser = argparse.ArgumentParser(
+        prog="CountCommandOutput",
+        description="Check Provided by GatorGrader: CountCommandOutput",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    # Required Named Checker Arguments {{{
+
+    required_group = parser.add_argument_group("required checker arguments")
+
+    # COMMAND: the command to execute
+    # REQUIRED? Yes
+    required_group.add_argument(
+        "--command", type=str, help="command to execute", required=True
+    )
+
+    # COUNT: the number of lines of output
+    # REQUIRED? Yes
+    required_group.add_argument(
+        "--count",
+        type=int,
+        metavar="COUNT",
+        help="how many of an entity should exist",
```

Comment:
I agree and have fixed this problem. I also found a few other spots where I could clarify the description of the command-line arguments &mdash; I went ahead and fixed those as well.

Thanks for pointing out this improvement! I should note that I tried to go through all of the command-line arguments in all of the checks and make then more uniform. For instance, in this example there is not, in fact, a need for the `metavar` since it is the same name as the parameter itself. I resolved this and several other types of similar issues.

I'm going to mark this as resolved. Let me know if there is any other way in which I can handle this issue, okay?